weepickle/test/src/com/rallyhealth/weepickle/v1/StructTests.scala

```diff
@@ -108,6 +104,11 @@ object StructTests extends TestSuite {
           Map[String, List[Int]](),
           "{}"
         )
+        test("non-string key") - {
+          TestUtil.rw(Map(new StringAnyVal("a") -> 1),
+            """{"a":1}"""
+          )
+        }
```

Comment:
It'd be nice to have an enum case as well.

Reply:
Thanks @adarshrhegde!